jwst/assign_wcs/miri.py

```diff
@@ -603,3 +603,32 @@ def get_wavelength_range(input_model, path=None):
     band = input_model.meta.instrument.band
 
     return dict([(ch + band, wr[ch + band]) for ch in channel])
+
+
+def store_dithered_position(input_model):
+    """Store the location of the dithered pointing
+    location in the wcsinfo
+
+    Parameters
+    ----------
+    input_model : jwst.datamodels.ImageModel
+        Data model containing dither offset information
+    pipeline : list
+        List of transforms to create a WCS object
+    """
+    # V2_ref and v3_ref should be in arcsec
+    idltov23 = IdealToV2V3(
+        input_model.meta.wcsinfo.v3yangle,
+        input_model.meta.wcsinfo.v2_ref, input_model.meta.wcsinfo.v3_ref,
+        input_model.meta.wcsinfo.vparity
+    )
+
+    dithered_v2, dithered_v3 = idltov23(input_model.meta.dither.x_offset, input_model.meta.dither.y_offset)
+
+    v23toworld = input_model.meta.wcs.get_transform('v2v3', 'world')
+    dithered_ra, dithered_dec, _ = v23toworld(dithered_v2, dithered_v3, 0.0)  # needs a wavelength,
+                                                                              # but does not affect return
+
+    input_model.meta.dither.dithered_ra = dithered_ra
+    input_model.meta.dither.dithered_dec = dithered_dec
+    return input_model
```

Comment:
Is it necessary to return the model? It only adds attributes and makes no copies.

Reply:
Good call, return statement has been removed.